src/js/close-button.js

```diff
@@ -12,7 +12,7 @@ import Component from './component';
  */
 class CloseButton extends Button {
 
- /**
+  /**
```

Comment:
I know this is WIP, but just thought I'd point out that this change has come up a few times. Not entirely sure what causes it.

Reply:
I saw that but seems like this fixes it? The `/` is one space back the function name. So, technically, all the `*` lines should be one space forward.

Reply:
yeah, Ideally everything is
```js
/**
 *
 *
 */
function foo() {}
```

But I don't really feel like fixing it up manually. We can just fix it forward whenever we see something misaligned.